### ISSUE TITLE
Preset/custom chord voicings

### ISSUE THREAD

**Issue by bhostetler18** [bhostetler18/tonality_features#3]
Allow saving certain shapes/voicings for quick access when creating a chord pad

**Comment by ghost** [comment]
Ive one more idea but this may sound crazy 🙃  
it'd be amazing to have playing styles or the possibility to save your own presets -and assign presets on chords you want-
with one chord pad preset, we could play each with various styles - hammered/arpeggio/rhodes/shuffle etc I and obv strum  ☺️
@bhostetler18

**Comment by bhostetler18** [comment]
@cazel-ea Good idea - could you make a new issue for this since it has to do with playback/MIDI rather than voicings?
